tests/config/nextflow.config

```diff
@@ -19,6 +19,7 @@ if ("$PROFILE" == "singularity") {
 } else {
     docker.enabled = true
     docker.userEmulation = true
+    docker.runOptions = "--platform linux/x86_64"
```

Comment:
Why does this need to be implicitly set?

Reply:
to make the biocontainers work on an `arm64` macbook 😅. This should have no effect on anything else

Reply:
Yes, sorry my question should have been whether this could have any other undesired effects, have not checked myself
If it doesn't then OK

Reply:
I tested this on both my mac and on our ubuntu server without any complaints, so I forsee no issues